portfolio/src/main/webapp/script.js

```diff
@@ -161,42 +194,47 @@ function fetchAndDisplayNumComments(num) {
     fetch('/data?num-comments='+num).then(response => response.json()).then((data) => {
         data.forEach((comment) => {
             dataListElement.appendChild(createCommentElement(comment));
+            displayMapComment(comment);
         });
     }); 
 }
 
 /** Creates a comment element by converting the object into Strings and concatenating them. */
 function createCommentElement(comment) {
-  const commentElement = document.createElement('li');
```

Comment:
Are these indentations intentional? (hard for me to tell with the Github diff tool since the indentation didn't change on previous line)

Reply:
The indentations look good to me on the file